### DISCUSSION TITLE
Add cheat to disable guests' fear of recently crashed rides

### DISCUSSION THREAD

**Discussion by Lunarexxy** [OpenRCT2/OpenRCT2#23295] [Feature requests]
I'm trying to build some repeating railguns and my ammunition keeps getting scared. Keeping their fear disabled as long as the cheat is active, instead of having to manually renew it, would help a lot.

This would also pair well with #16788

**Comment by fidwell** [comment]
Is that not what the "Reset crash status" button in the cheat menu does?

**Reply by Lunarexxy** [reply]
When you click it, yes. This is a request for a cheat that would be constantly active, without having to press the button every time.

**Comment by ZehMatt** [comment]
I think this is too niche to have built-in. It should be fairly easy to create a plugin that just uses the game action to reset the crash status periodically.

**Reply by Crazycolbster** [reply]
I agree. It should be pretty easy to set up a listener that looks for any crash event, and instantly reset crash status whenever it activates
